src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/NegotiateResponseTest.java

```diff
@@ -5,10 +5,11 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-import com.google.gson.stream.JsonReader;
+import java.io.StringReader;
+
 import org.junit.jupiter.api.Test;
 
-import java.io.StringReader;
```

Comment:
Is there some ordering convention that we're using for the imports?

Reply:
Yeah, java, javax, org, com enforced by this rule https://github.com/aspnet/AspNetCore/blob/1f63f25508dfb8f46715d1842cf3dd49bd21a300/src/SignalR/clients/java/signalr/build.gradle#L47